### PR TITLE
Fix/check initialised

### PR DESCRIPTION
**Pull request by albertjan**
throws an error message if a function is called before processing is initialised.

See: http://albertjan-dev.trinket.io/python/9f0110db88

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -67,7 +69,11 @@ export function makeFunc(thingToWrap, name, args_template) {
     let jsfunc = function wrappedFunc() {
         let functionToWrap = null;
 
-        if (typeof thingToWrap != "function") {
+        if (typeof thingToWrap !== "function") {
+            if (!thingToWrap[__isinitialised__]) {
+                throw new Error(`cannot call "${name}" outside "draw", "setup" or event handlers`);
```
Comment (albertjan) [inline src/utils.js]: @eah13 you might want to think about the wording of this error. And we might want to throw a real python exception here. But which one?

**Review by brianpmarks — COMMENTED** [review]
Looks good to me! There are conflicts on the built files though so maybe previous PRs need to be pulled into this branch?

**Comment by albertjan** [comment]
Yeah I'll merge and rebuild.

File: src/utils.js (hunk 1 above)
Comment (eah13) [reply src/utils.js:74]: > throw a real python exception

I can't think of an appropriate one.  I don't think you'd encounter an analogous situation in Python.

We could invent a new exception ala `SuspensionError`, perhaps `ProcessingError`?

**Pull request merged**